### ISSUE TITLE
Lava Berries: directly into lava in machines which take lava.

### ISSUE THREAD

**Issue by Divineaspect** [ganymedes01/Gany-s-Nether#40]
I'm primarily thinking of the magmatic centrifuge, but this would be broadly helpful.

**Comment by Blue64** [comment]
I believe that the simplest way would be to register them as a Lava Container, thus draining their "Juice" (lava), and potentially returning an empty Berry that you can throw into the Organic Compresser or something (Ex Nihilo compost works too! lol)

**Comment by buttilda** [comment]
Yeah it wouldn't be too hard to implement, but I don't like the idea.
Removing steps just to make things simpler makes the game boring )=

**Comment by Divineaspect** [comment]
I'd be quite happy with a gany's nether bases squeezer, I don't care if it's easier, but rather that I don't need other mods. And you don't have an autocrafter.

**Comment by buttilda** [comment]
but without other mods you wouldn't be able to automatically harvest the berries :3

**Comment by Divineaspect** [comment]
but I'm using Gany's surface for it...

**Comment by buttilda** [comment]
ah... right, forgot about that XD
Ok, I'll think of something :3

**Comment by MagmaSlime123** [comment]
> Yeah it wouldn't be too hard to implement, but I don't like the idea. Removing steps just to make things simpler makes the game boring )=

Remove your mods if you no longer care about them.
